packages/office-ui-fabric-react/src/components/ActivityItem/examples/ActivityItem.Basic.Example.tsx

```diff
@@ -1,15 +1,27 @@
 import * as React from 'react';
-import { css, classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
-import { ActivityItem } from '../ActivityItem';
-import { getStyles, IActivityItemExampleStyleProps, IActivityItemExampleStyles } from './ActivityItem.Example.styles';
-import { Icon } from '../../../Icon';
-import { Link } from '../../../Link';
+import { ActivityItem, css, classNamesFunction, Icon, IStyle, Link } from 'office-ui-fabric-react';
+
+type IActivityItemExampleStyleProps = {};
+
+interface IActivityItemExampleStyles {
```

Comment:
Do we really need these interfaces, or is it okay to just use `mergeStyleSets` on the `exampleStyles` object?

Reply:
I copied the code over as it was and made quite a few improvements already. I'd like to cut off further improvements for this PR as I have quite a lot of other issues to get to from shield.

Reply:
Fair enough!